hooli-data-eng/hooli_data_eng/defs/dbt/component.yaml

```diff
@@ -0,0 +1,34 @@
+type: hooli_data_eng.defs.dbt.component.HooliDbtComponent
+
+attributes:
+  groups:
+    - selection: "company_stats locations_cleaned sku_stats company_perf"
+      run_on_code_version_change: true
+
+    - selection: "weekly_order_summary order_stats"
+      partitioning: "weekly"
+      freshness_check_sensor: weekly_freshness_check_sensor
+
+    - selection: "orders_cleaned users_cleaned orders_augmented"
+      partitioning: "daily"
+
+---
+
+type: hooli_data_eng.components.ScheduledJobComponent
+
+attributes:
+  job_name: dbt_components_job
+  cron_schedule: "0 8 * * *"
+  asset_selection: 'tag:"all_cleaned_models"'
```

Comment:
obviously this was my code not yours, but @alangenfeld  is there any way to avoid the slightly ugly single quote double quote thing for asset selection? Not the end of the world but it annoyed me when I was writing it

Reply:
ya i just copied it but looks like its still valid if you omit it